tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs

```diff
@@ -39,6 +39,26 @@ internal static class Debugger
             /// <seealso cref="DebuggerSettings.UploadBatchSize"/>
             public const string UploadBatchSize = "DD_DEBUGGER_UPLOAD_BATCH_SIZE";
 
+            /// <summary>
+            /// Configuration key for the maximum symbol size to upload (in bytes).
+            /// Default value is 1 mb.
+            /// </summary>
+            /// <seealso cref="DebuggerSettings.SymbolBatchSizeInBytes"/>
+            public const string SymbolBatchSizeInBytes = "DD_DEBUGGER_SYMBOL_BATCH_SIZE_BYTES";
```

Comment:
as per @GreenMatan feedback we need to decide whether to change this for consistency

Reply:
I would say we should. 👍

Reply:
Fixed in d76bac0e9bd3cfbda3e480deff9bbd0e453a32ec

Reply:
You have to include the new environment variables to a file named `config_norm_rules.json` (otherwise the CI won't pass) and you have to introduce new PR to dd-go as Andrew pointed out in Slack.

Reply:
Fixed in 4cf1caa7a6a45f9daaedc7d4cb34e6954a57cc51